### ISSUE TITLE
Update furo wheel package to the latest 2022.9.29

### ISSUE THREAD

**Issue by kwankyu** [sagemath/sage#34656]
<div id="comment:0"></div>

There is little visible difference from the previous version, except

- Change right sidebar title from "Contents" to "On this page"

I expected style updates to make <h2> headings look more separated from <section> elements, but no such updates. 

Instead, we may put a thin rule below, for example, "Mathematics", but this is a work for another ticket.

Depends on #34434

CC:  @mkoeppe

Component: **packages: standard**

Author: **Kwankyu Lee**

Branch/Commit: **[`107b88e`](https://github.com/sagemath/sagetrac-mirror/commit/107b88e501af2dd436a65f2ff7f0563b7faed5ff)**

Reviewer: **Matthias Koeppe**

_Issue created by migration from https://trac.sagemath.org/ticket/34656_

**Comment by kwankyu** [comment]
Branch: **[u/klee/update_furo_to_2022_9_29](https://github.com/sagemath/sagetrac-mirror/tree/u/klee/update_furo_to_2022_9_29)**

**Comment by kwankyu** [comment]
Commit: **[`c6b16a2`](https://github.com/sagemath/sagetrac-mirror/commit/c6b16a26f78e0f359a0009f47f798bcd12226448)**

**Comment by kwankyu** [comment]
Author: **Kwankyu Lee**

**Comment by kwankyu** [comment]
<div id="comment:2"></div>

New commits:
<table><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/c6b16a26f78e0f359a0009f47f798bcd12226448"><code>c6b16a2</code></a></td><td><code>Update furo wheel package to 2022.9.29</code></td></tr></table>

**Comment by kwankyu** [comment]
<div id="comment:3" align="right">comment:3</div>

But I get

```
Using cached file /Users/kwankyu/GitHub/sage-dev/upstream/furo-2022.9.29-py3-none-any.whl
furo-2022.9.29
====================================================
Setting up build directory for furo-2022.9.29
****************************************************
Host system:
Darwin Hera.local 21.6.0 Darwin Kernel Version 21.6.0: Mon Aug 22 20:17:10 PDT 2022; root:xnu-8020.140.49~2/RELEASE_X86_64 x86_64
****************************************************
C compiler: gcc
C compiler version:
Apple clang version 14.0.0 (clang-1400.0.29.102)
Target: x86_64-apple-darwin21.6.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
****************************************************
Uninstalling existing 'furo'
Running pip-uninstall script for 'furo'
Found existing installation: furo 2022.6.21
Uninstalling furo-2022.6.21:
  Successfully uninstalled furo-2022.6.21
Removing stamp file '/Users/kwankyu/GitHub/sage-dev/local/var/lib/sage/venv-python3.10/var/lib/sage/installed/furo-2022.6.21'
WARNING: Skipping furo as it is not installed.
Using pip 22.2.2 from /Users/kwankyu/GitHub/sage-dev/local/var/lib/sage/venv-python3.10/lib/python3.10/site-packages/pip (python 3.10)
Looking in links: /Users/kwankyu/GitHub/sage-dev/local/var/lib/sage/venv-python3.10/var/lib/sage/wheels
Processing ./inst/Users/kwankyu/GitHub/sage-dev/local/var/lib/sage/venv-python3.10/var/lib/sage/wheels/furo-2022.9.29-py3-none-any.whl
ERROR: Wheel 'furo' located at /Users/kwankyu/GitHub/sage-dev/local/var/lib/sage/venv-python3.10/var/tmp/sage/build/furo-2022.9.29/inst/Users/kwankyu/GitHub/sage-dev/local/var/lib/sage/venv-python3.10/var/lib/sage/wheels/furo-2022.9.29-py3-none-any.whl is invalid.
Warning: installing with "python3 -m pip install --verbose --no-index --find-links=/Users/kwankyu/GitHub/sage-dev/local/var/lib/sage/venv-python3.10/var/lib/sage/wheels --disable-pip-version-check --isolated --no-cache-dir" failed. Retrying, adding "--no-deps --ignore-installed --ignore-requires-python"
Using pip 22.2.2 from /Users/kwankyu/GitHub/sage-dev/local/var/lib/sage/venv-python3.10/lib/python3.10/site-packages/pip (python 3.10)
Looking in links: /Users/kwankyu/GitHub/sage-dev/local/var/lib/sage/venv-python3.10/var/lib/sage/wheels
Processing ./inst/Users/kwankyu/GitHub/sage-dev/local/var/lib/sage/venv-python3.10/var/lib/sage/wheels/furo-2022.9.29-py3-none-any.whl
ERROR: Wheel 'furo' located at /Users/kwankyu/GitHub/sage-dev/local/var/lib/sage/venv-python3.10/var/tmp/sage/build/furo-2022.9.29/inst/Users/kwankyu/GitHub/sage-dev/local/var/lib/sage/venv-python3.10/var/lib/sage/wheels/furo-2022.9.29-py3-none-any.whl is invalid.
Error: installing with pip  failed
********************************************************************************
Error installing furo-2022.9.29-py3-none-any.whl
********************************************************************************     
```

**Comment by mkoeppe** [comment]
<div id="comment:7" align="right">comment:7</div>

What commands did you use to do this update?

**Comment by kwankyu** [comment]
<div id="comment:8" align="right">comment:8</div>

`sage --package update-latest furo`

**Comment by mkoeppe** [comment]
<div id="comment:9" align="right">comment:9</div>

OK, I'll make this command ready for wheel packages.

**Comment by mkoeppe** [comment]
<div id="comment:10" align="right">comment:10</div>

This will happen in #34434

**Comment by kwankyu** [comment]
<div id="comment:11" align="right">comment:11</div>

Thanks.

**Comment by 7ed8c4ca-6d56-4ae9-953a-41e42b4ed313** [comment]
<div id="comment:12"></div>

Branch pushed to git repo; I updated commit sha1. New commits:
<table><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/73055b1f7de4955f5143c2a477a59d4ea945637a"><code>73055b1</code></a></td><td><code>sage -package update-latest: Handle wheel packages, use distribution name from install-requires.txt</code></td></tr><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/b0f638d9b3db3e2a251d530217e8c7d090bce860"><code>b0f638d</code></a></td><td><code>Minor fixes</code></td></tr><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/b4481349652f3a4bc7f4890c7f05730a7ac49cbc"><code>b448134</code></a></td><td><code>Merge branch 't/34434/_sage__package_update_latest__should_read_install_requires_txt_and_handle_wheel_packages' into t/34656/update_furo_to_2022_9_29</code></td></tr><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/107b88e501af2dd436a65f2ff7f0563b7faed5ff"><code>107b88e</code></a></td><td><code>Update furo wheel package correctly to 2022.9.29</code></td></tr></table>

**Comment by 7ed8c4ca-6d56-4ae9-953a-41e42b4ed313** [comment]
Changed commit from **[`c6b16a2`](https://github.com/sagemath/sagetrac-mirror/commit/c6b16a26f78e0f359a0009f47f798bcd12226448)** to **[`107b88e`](https://github.com/sagemath/sagetrac-mirror/commit/107b88e501af2dd436a65f2ff7f0563b7faed5ff)**

**Comment by kwankyu** [comment]
Dependencies: **#34434**

**Comment by kwankyu** [comment]
Description changed:
``````diff
--- 
+++ 
@@ -1 +1,7 @@
+There is no big visible difference from the previous version.
 
+I expected style updates to make <h2> headings look more separated from <section> elements, but in vain. 
+
+We may put a thin rule below, for example, "Mathematics", but this is a work for another ticket... 
+
+
``````

**Comment by kwankyu** [comment]
Description changed:
``````diff
--- 
+++ 
@@ -1,7 +1,9 @@
-There is no big visible difference from the previous version.
+There is little visible difference from the previous version, except
 
-I expected style updates to make <h2> headings look more separated from <section> elements, but in vain. 
+- Change right sidebar title from "Contents" to "On this page"
 
-We may put a thin rule below, for example, "Mathematics", but this is a work for another ticket... 
+I expected style updates to make <h2> headings look more separated from <section> elements, but no such updates. 
+
+Instead, we may put a thin rule below, for example, "Mathematics", but this is a work for another ticket.
 
 
``````

**Comment by mkoeppe** [comment]
Reviewer: **Matthias Koeppe**

**Comment by kwankyu** [comment]
<div id="comment:18" align="right">comment:18</div>

Thanks!

**Comment by vbraun** [comment]
Changed branch from **[u/klee/update_furo_to_2022_9_29](https://github.com/sagemath/sagetrac-mirror/tree/u/klee/update_furo_to_2022_9_29)** to **[`107b88e`](https://github.com/sagemath/sagetrac-mirror/commit/107b88e501af2dd436a65f2ff7f0563b7faed5ff)**
